src/main/java/com/icthh/xm/ms/balance/service/mapper/BalanceMapper.java

```diff
@@ -8,10 +8,11 @@
 /**
  * Mapper for the entity Balance and its DTO BalanceDTO.
  */
-@Mapper(componentModel = "spring", uses = {})
+@Mapper(componentModel = "spring")
 public interface BalanceMapper extends EntityMapper<BalanceDTO, Balance> {
 
-
+    @Mapping(target = "removePockets", ignore = true)
+    @Mapping(target = "removeMetrics", ignore = true)
```

Comment:
please explain why this new annotations is needed

Reply:
Same as above